source/_lovelace/conditional.markdown

```diff
@@ -22,33 +22,41 @@ conditions:
       required: true
       description: Home Assistant entity ID.
       type: string
-    state:
-      required: false
-      description: Entity state is equal to this value.*
+    operator:
+      required: true
+      description: Operator to use in the comparison. Can be `==`, `<=`, `<`, `>=`, `>`, `!=`, `in`, `not in`, or `regex`.
       type: string
-    state_not:
+    value:
+      required: true
+      description: Value which will be compared with the entity state.
+      type: string
```

Comment:
not sure about type. It can be string but also number, ..
But in the entity filter card is also only type string.